Roll for Shoes/sheet.html

```diff
@@ -26,46 +26,46 @@ <h3>Statuses</h3>
 </div>
 
 <script type="text/worker">
-  function useAttr(key, cb) {
-    useAttrs([key], (attrs) => {
-      cb(attrs.at(0))
-    })
-  }
+    function useAttr(key, cb) {
+      useAttrs([key], (attrs) => {
+        cb(attrs.at(0))
+      })
+    }
 
-  function useAttrs(keys, cb) {
-    getAttrs(keys, (attrs) => {
-      cb(Object.values(attrs))
-    })
-  }
+    function useAttrs(keys, cb) {
+      getAttrs(keys, (attrs) => {
+        cb(Object.values(attrs))
+      })
+    }
 
-  on('clicked:repeating_skills:skillcheck', () => {
-    useAttrs(['repeating_skills_name', 'repeating_skills_rating'], ([skillName, ratingString]) => {
-      const rating = parseInt(ratingString)
-      if (rating === 0) return
+      on('clicked:repeating_skills:skillcheck', () => {
+      useAttrs(['repeating_skills_name', 'repeating_skills_rating'], ([skillName, ratingString]) => {
+          const rating = parseInt(ratingString);
+          if (rating === 0) return;
 
-      const rollQuery = rating > 0
-        ? `${rating}d6`
-        : `${Math.abs(rating)}d6kl1`
+          let rollTemplate = `&{template:default} {{name=${skillName}}}`
 
-      startRoll(`&{template:default} {{name=${skillName} ${rating}}} {{result=[[${rollQuery}+?{Modifier}]]}}`, ({ rollId }) => {
-        finishRoll(rollId)
+          const modifier = `+?{Modifier|0}`;
+
+          for (let i = 1; i <= Math.abs(rating); i++) {
+              rollTemplate += ` {{roll${i}=[[1d6${modifier}]]}}`
```

Comment:
Doesn't this add a modifier for *each* die rolled? That's not how statuses and modifiers should work in the system.

Reply:
This also removes the possibility of negative skills, which are an option (as Flaws) and should remain possible.